core/DataTable/Filter/ColumnDelete.php

```diff
@@ -160,11 +160,15 @@ protected function removeColumnsFromTable(&$table)
                 continue;
             }
             foreach ($this->columnsToRemove as $column) {
-
-                if (!array_key_exists($column, $row)) {
-                    continue;
+                if (is_object($row)) {
```

Comment:
@sgiehl I reckon it's not working maybe because we need to use indeed `isset()` or so as in the row object  it might not have the property... 
wondering if we even need this code at all as maybe unset would do the work anyway no matter if the property/key is set?

Reply:
We couldn't remove it actually.